pom.xml

```diff
@@ -72,28 +69,39 @@
 			<artifactId>aws-cloudwatch-library</artifactId>
 			<version>1.11.136</version>
 			<optional>true</optional>
+			<exclusions>
+				<!-- Provided by apache-httpcomponents-client-4-api plugin -->
+				<exclusion>
+					<groupId>org.apache.httpcomponents</groupId>
+					<artifactId>httpclient</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.apache.httpcomponents</groupId>
+					<artifactId>httpcore</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<groupId>org.jenkins-ci.plugins</groupId>
+			<artifactId>apache-httpcomponents-client-4-api</artifactId>
+			<version>4.5.14-150.v7a_b_9d17134a_5</version>
+			<optional>true</optional>
+		</dependency>
+		<dependency>
+			<groupId>org.jenkins-ci.plugins</groupId>
+			<artifactId>bouncycastle-api</artifactId>
+			<version>2.28</version>
+			<optional>true</optional>
 		</dependency>
 	</dependencies>
 
 	<scm>
-		<connection>scm:git:ssh://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
-		<developerConnection>scm:git:ssh://git@github.com/jenkinsci/${project.artifactId}-plugin.git</developerConnection>
-		<url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
-		<tag>HEAD</tag>
+		<connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
+		<developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
+		<url>https://github.com/${gitHubRepo}</url>
+		<tag>${scmTag}</tag>
```

Comment:
for mvn release, I would prefer not to change that

Reply:
I am not going to argue with you about this. This is standard practice for all Jenkins plugins as in https://github.com/jenkinsci/archetypes.